pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -101,7 +101,12 @@ type invalidPlugins struct {
 var invalidPluginsByVersion = []invalidPlugins{
 	{
 		schemeGroupVersion: v1.SchemeGroupVersion.String(),
-		plugins:            []string{},
```

Comment:
Plz add a test to validate this, thanks. Otherwise LGTM

Reply:
@kerthcet  added.